content/blog/posts/o3de-introducing-design-system.md

```diff
@@ -0,0 +1,95 @@
+
+So to further ensure that the editor upheld these high quality of these standards, we developed some tenets that contributors should follow:
+
+![Diagram showing the tenets of the O3DE design system](/images/blog/intro-to-bjds/tenets-diagram.jpg)
+
+* **Efficiency:** User interactions and workflows should value efficiency and ease of use.
+* **Accessibility:** Accessibility options, such as color contrasts, fonts and icon legibility, interactions and scalability etc. are built into our the tool from the first step.
+* **Familiarity:** Align to our standards of the visual interfaces, tools, branding, and UI elements. No need to reinvent the wheel and confuse users.
+* **Modularity:** The key to each element of O3DE, including the interface design and component development.
+
+These tenets were formed to be straightforward and easy to remember. We want to ensure that the BJDS tenets are flexible for the community to embrace, and to employ through their contributions to the engine.
+
+### What's included in the BlueJay Design System
+
+**The BlueJay Design System** is made up of a set of standardized components and layouts to help accelerate the creation of new workflows for the editor. It also has [supporting documentation](https://o3de.org/docs/tools-ui/) on the O3DE Learn website, which details the following:
+
+#### UI Components
+
+UI components are the various individual widgets that make up the editor experience. The BJDS documentation outlines details like visual markup properties, rules around how to use them and guides when to use them in which scenario. Currently, with [20 UI components](https://o3de.org/docs/tools-ui/component-library/) detailed (and more coming), these include code samples and API references to integrate into your editor extension.
+
+If you are looking for information on key building blocks that O3DE uses, [UI Components](https://o3de.org/docs/tools-ui/component-library/) is the right place to read. It's the systematic reuse of these components that make up the visual and functional consistency across O3DE.
+
+![Preview @mdash; for the O3DE UI component guidelines](/images/blog/intro-to-bjds/ui-components-preview.jpg)
+
+#### UX Patterns
+
+UX Patterns are a series of best practice solutions that a designer or developer should follow when creating new workflows and components in the editor. Unlike UI Components, where it's mostly about the visual aesthetics and how it mechanically operates, UX Patterns is more about how a user interacts and behaves for a task with a goal in mind.
+
+For e.g., the anatomy of a component card and it's different types can be found on the [UI Component > Card page](https://o3de.org/docs/tools-ui/component-library/uidev-card-component/), while the what contents go in the card, how a user interacts with it and how O3DE interacts with the card itself can be found on the [UX Patterns > Component Card UX Patterns](https://o3de.org/docs/tools-ui/ux-patterns/component-card/overview/) page.
```

Comment:
- Consider splitting the sentence since it's a bit long. 
- Fixed the link. 

```suggestion
For example, the anatomy of a component card and it's different types can be found in the [UI Component > Card](https://o3de.org/docs/tools-ui/component-library/uidev-card-widget/) page. What contents go in the card, how a user interacts with it and how O3DE interacts with the card itself can be found in the [UX Patterns > Component Card UX Patterns](https://o3de.org/docs/tools-ui/ux-patterns/component-card/overview/) page.
```